scene/resources/syntax_highlighter.cpp

```diff
@@ -209,7 +227,11 @@ Dictionary CodeHighlighter::_get_line_syntax_highlighting_impl(int p_line) {
 								}
 							}
 							prev_color = color_regions[in_region].color;
+							prev_style = color_regions[in_region].style;
+							prev_text_segment = color_regions[in_region].text_segment;
 							highlighter_info["color"] = color_regions[c].color;
+							highlighter_info["style"] = color_regions[c].style;
+							highlighter_info["text_segment"] = color_regions[c].text_segment;
```

Comment:
Can you update the docs for `syntax_highlighter.xml` for `get_line_syntax_highlighting`? It only mentions the `color` option. Would be good to mention other overrides